azext_iot/tests/digitaltwins/test_dt_data_history_lifecycle_int.py

```diff
@@ -0,0 +1,481 @@
+                instance_name,
+                self.rg,
+                "t",
+                ADX_DATABASE,
+                ADX_RG,
+                ADX_CLUSTER,
+                EP_EVENTHUB_NAMESPACE,
+                EP_EVENTHUB_TOPIC,
+                EP_RG,
+            ),
+            expect_failure=True
+        )
+
+        self.cmd(
+            "dt data-history connection create adx -n {} -g {} --cn {} --adxd {} --adxg {} "
+            "--adxc {} --ehn {} --eh {} --ehg {} -y".format(
+                instance_name,
+                self.rg,
+                connection_name,
+                "testresource",
+                ADX_RG,
+                ADX_CLUSTER,
+                EP_EVENTHUB_NAMESPACE,
+                EP_EVENTHUB_TOPIC,
+                EP_RG,
+            ),
+            expect_failure=True
+        )
+
+        self.cmd(
+            "dt data-history connection create adx -n {} -g {} --cn {} --adxd {} --adxg {} "
+            "--adxc {} --ehn {} --eh {} --ehg {} -y".format(
+                instance_name,
+                self.rg,
+                connection_name,
+                ADX_DATABASE,
+                ADX_RG,
+                "testresource",
+                EP_EVENTHUB_NAMESPACE,
+                EP_EVENTHUB_TOPIC,
+                EP_RG,
+            ),
+            expect_failure=True
+        )
+
+        self.cmd(
+            "dt data-history connection create adx -n {} -g {} --cn {} --adxd {} --adxg {} "
+            "--adxc {} --ehn {} --eh {} --ehg {} -y".format(
+                instance_name,
+                self.rg,
+                connection_name,
+                ADX_DATABASE,
+                ADX_RG,
+                ADX_CLUSTER,
+                "testresource",
+                EP_EVENTHUB_TOPIC,
+                EP_RG,
+            ),
+            expect_failure=True
+        )
+
+        self.cmd(
+            "dt data-history connection create adx -n {} -g {} --cn {} --adxd {} --adxg {} "
+            "--adxc {} --ehn {} --eh {} --ehg {} -y".format(
+                instance_name,
+                self.rg,
+                connection_name,
+                ADX_DATABASE,
+                ADX_RG,
+                ADX_CLUSTER,
+                EP_EVENTHUB_NAMESPACE,
+                "testresource",
+                EP_RG,
+            ),
+            expect_failure=True
+        )
+
+        self.cmd(
+            "dt data-history connection create adx -n {} -g {} --cn {} --adxd {} --adxg {} "
+            "--adxc {} --ehn {} --eh {} --ehg {} --ehc {} -y".format(
+                instance_name,
+                self.rg,
+                connection_name,
+                ADX_DATABASE,
+                ADX_RG,
+                ADX_CLUSTER,
+                EP_EVENTHUB_NAMESPACE,
+                EP_EVENTHUB_TOPIC,
+                EP_RG,
+                "testresource",
+            ),
+            expect_failure=True
+        )
+
+    def test_dt_data_history_adx_wait(self):
```

Comment:
Separating out this test into another test script, will allow the test to be executed in parallel with the test_dt_data_history_adx test. This should significantly reduce the total time to complete the tests.

Reply:
Will keep in the same script for now